flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/batch/connectors/cassandra/CustomCassandraAnnotatedPojo.java

```diff
@@ -21,11 +21,9 @@
 import com.datastax.driver.mapping.annotations.Table;
 
 /** Example of Cassandra Annotated POJO class for use with {@link CassandraPojoInputFormat}. */
-@Table(name = CustomCassandraAnnotatedPojo.TABLE_NAME, keyspace = "flink")
+@Table(name = "batches", keyspace = "flink")
```

Comment:
we should be able to just remove this (and other) annotations because we override them anyway.

Reply:
I agree thanks: as we will now add the annotation rather than modifying the existing one. It will be less confusing